### ISSUE TITLE
Can we use monaco-loader or approach to instead of using CDN?

### ISSUE THREAD

**Issue by knoxai** [arshad-yaseen/monacopilot#5]

**Comment by arshad-yaseen** [comment]
Could you please elaborate on what you mean? We are using [@monaco-editor/react](https://www.npmjs.com/package/@monaco-editor/react) for the editor. They use monaco-loader to load the editor.

**Comment by knoxai** [comment]
> Could you please elaborate on what you mean? We are using [@monaco-editor/react](https://www.npmjs.com/package/@monaco-editor/react) for the editor. They use monaco-loader to load the editor.

I meant can we not use CDN(cdn.jsdelivr.net) or how do we switch it to local loader like [@monaco-editor/loader](https://github.com/suren-atoyan/monaco-loader). cdn.jsdelivr.net is great but not for all Countries. some users just keep seeing Loading... all the time while open a file with editor.

**Comment by arshad-yaseen** [comment]
I gotcha!

We released `v0.8.22` 🎉

You can now import (named) `loader` the same way as from `@monaco-editor/react`. To update, run:

```bash
npm update monacopilot
```

Then, import and configure `loader`:

```jsx
import * as monaco from 'monaco-editor';
import { loader } from 'monacopilot';

loader.config({
  monaco,
});

// or

loader.config({
  paths: {
    vs: "..."
  }
});
```

For more details on loader configuration, refer to the [@monaco-editor/react loader config documentation](https://github.com/suren-atoyan/monaco-react?tab=readme-ov-file#loader-config)

**Comment by knoxai** [comment]
Fantastic! Thank You!
